tracetools_launch/tracetools_launch/action.py

```diff
@@ -113,10 +113,11 @@ def __init__(
         self,
         *,
         session_name: SomeSubstitutionsType,
+        is_init_session: Union[bool, SomeSubstitutionsType] = False,
```

Comment:
I think we should call this `snapshot_session` session, since it could technically be used not _just_ for init events: you could use `is_snapshot_session=True` and `events_ust=DEFAULT_EVENTS_ROS` to do a snapshot with all events. In that case, the snapshot being used for init events is just 1 use case, even though it is probably the main use case (hence why we set `events_ustDEFAULT_INIT_EVENTS_ROS` if `snapshot_session`).